### ISSUE TITLE
Improve Styling for a better accessibility

### ISSUE THREAD

**Issue by 00travelgirl00** [WordPress/health-check#295]
Currently the font-size is 13px, which is pretty small and can be hard to read for people with visual impairment.

Also links at the details are not underlined which make it hard (or even impossible) for colorblind people to spot the link.

I suggest to increase the font-size to 15px and to underline the links, which are surrounded by text. For the underline I would suggest:

https://plugins.trac.wordpress.org/ticket/2772

**Comment by 00travelgirl00** [comment]
<img width="1438" alt="larger_font-size underline-links-black white" src="https://user-images.githubusercontent.com/21318377/53955354-7d913b80-40e1-11e9-986d-fe3520a00173.png">
<img width="1438" alt="larger_font-size underlined_links" src="https://user-images.githubusercontent.com/21318377/53955355-7e29d200-40e1-11e9-8c93-33ffc99a7717.png">
<img width="1439" alt="small_font_size_no_underline" src="https://user-images.githubusercontent.com/21318377/53955358-7e29d200-40e1-11e9-9790-d402be1909aa.png">
<img width="1439" alt="small_fontsize no_underline_black white" src="https://user-images.githubusercontent.com/21318377/53955359-7e29d200-40e1-11e9-8141-be63900831b2.png">

**Comment by Clorith** [comment]
The revamped UX and UI in #227 has improved readability by implementing underlines and larger fonts (the equivalent font size of 16px) than the rest of the wp-admin area for improved usability.

As this has been put into the development version at this time, I'm closing this ticket 👍
